sdk/python/foundation-models/system/docs/component_docs/nlp_finetune/finetune_component_parameters.md

```diff
@@ -0,0 +1,210 @@
+# Finetune Pipeline Component
+This component enables finetuning of pretrained models on custom or pre-available datasets. The component supports Deepspeed for performance enhancement. The component supports parameter efficient finetuning with techniques like LoRA.
```

Comment:
Shall we add more optimizations that we support today? multi-gpu, multi-node, mixed precision, flash attention as well?